### PR TITLE
mobile: DO NOT MERGE test swift slowness

### PR DESCRIPTION
**Pull request by abeyad**

### CODE REVIEW

File: .github/workflows/mobile-ios_tests.yml [hunk 1]
```diff
@@ -57,7 +57,8 @@ jobs:
         - name: Run swift library tests
           args: >-
             test
-            --config=mobile-remote-ci-macos-ios-swift
+            --config=mobile-remote-ci-macos
```
Comment (phlax) [inline .github/workflows/mobile-ios_tests.yml:60]: @abeyad this wont be noticed in a PR - you can use staging if you needed

**Comment by adisuissa** [comment]
Assigning @abeyad as this is not for submission.
/assign @abeyad

**Comment by adisuissa** [comment]
Assigning myself to remove Slack-bot notification
/assign @adisuissa

**Pull request closed without merging**